### ISSUE TITLE
 java.lang.IncompatibleClassChangeError when running dependency-check-gradle 10.0.4 in gradle 7.6.4

### ISSUE THREAD

**Issue by Grimoren** [dependency-check/dependency-check-gradle#411]
```
...
         Caused by: java.lang.IncompatibleClassChangeError: Class org.gradle.api.internal.artifacts.configurations.DefaultConfiguration_Decorated does not implement the requested interface org.gradle.api.Named
         at org.owasp.dependencycheck.gradle.tasks.AbstractAnalyze.shouldBeSkipped(AbstractAnalyze.groovy:334)
         at org.owasp.dependencycheck.gradle.tasks.AbstractAnalyze$_processConfigurations_closure7.doCall(AbstractAnalyze.groovy:431)
         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
         at org.gradle.api.specs.internal.ClosureSpec.isSatisfiedBy(ClosureSpec.java:33)
         at org.gradle.api.internal.collections.CollectionFilter.filter(CollectionFilter.java:46)
         at org.gradle.api.internal.collections.FilteredCollection$FilteringIterator.findNext(FilteredCollection.java:122)
         at org.gradle.api.internal.collections.FilteredCollection$FilteringIterator.<init>(FilteredCollection.java:116)
         at org.gradle.api.internal.collections.FilteredCollection.iterator(FilteredCollection.java:161)
         at org.gradle.api.internal.DefaultDomainObjectCollection.findAll(DefaultDomainObjectCollection.java:448)
         at org.gradle.api.internal.DefaultNamedDomainObjectSet.findAll(DefaultNamedDomainObjectSet.java:82)
         at org.owasp.dependencycheck.gradle.tasks.AbstractAnalyze.processConfigurations(AbstractAnalyze.groovy:430)
         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
         at org.owasp.dependencycheck.gradle.tasks.Aggregate$_scanProject_closure1.doCall(Aggregate.groovy:56)
         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
...




```

**Comment by vidgeus** [comment]
@jeremylong is there an update on this issue ? We are facing the same error using gradle 7.6.4 and latest OWASP (`10.0.4`)
```build.gradle
buildscript {
    dependencies {
        classpath 'org.owasp:dependency-check-gradle:+'
    }
}
```
**NOTE:** currently on this GitHub repo it says latest release is `10.0.3`, but on the plugin page ([link](https://plugins.gradle.org/plugin/org.owasp.dependencycheck)) it says latest is `10.0.4`.

**Comment by jeremylong** [comment]
This should be resolved with the release of 11.0.0.
